src/iso.js

```diff
@@ -1,5 +1,14 @@
 const KEY_NAME = 'data-iso-key'
 
+const rLt = /</g
+const rGt = />/g
+const rLte = /&lt;/g
+const rGte = /&gt;/g
+const rEnc = /[<>]/
+const rDec = /&lt;|&gt;/
+
+const toString = val => val ? String(val) : ''
```

Comment:
Yes it is. It's mostly for undefined but this was shorter to write and the other cases don't matter 
much because the type is (object or undefined)